bin/gwdetchar-slow-correlation

```diff
@@ -295,6 +300,7 @@ def process_channel(input_,):
 
 pool = multiprocessing.Pool(nprocplot)
 results = pool.map(process_channel, auxdata.iteritems())
+#results = pool.map(partial(process_channel, thresh=threshold), auxdata.iteritems())
```

Comment:
I removed it by redoing those changes (easy now but I know it will be a bad idea to do that later), and i will use that next time.